TAO/tao/Strategies/DIOP_Acceptor.cpp

```diff
@@ -1038,6 +1071,20 @@ TAO_DIOP_Acceptor::parse_options (const char *str)
                                  ACE_TEXT ("endpoint priorities no longer supported.\n")),
                                 -1);
             }
+          else if (name == "portspan")
+            {
+              int range = static_cast <int> (ACE_OS::atoi (value.c_str ()));
+              // @@ What's the lower bound on the range?  zero, or one?
+              if (range < 1 || range > ACE_MAX_DEFAULT_PORT)
+                TAOLIB_ERROR_RETURN ((LM_ERROR,
+                                   ACE_TEXT ("TAO (%P|%t) Invalid IIOP endpoint ")
```

Comment:
DIOP

Reply:
Fixed. My change will show when I push the next commit.